plugins/backstage-plugin-coder/src/components/CoderAuthWrapper/CoderAuthWrapper.test.tsx

```diff
@@ -166,8 +166,6 @@ describe(`${CoderAuthWrapper.name}`, () => {
 
         unmount();
       }
-
-      expect.hasAssertions();
```

Comment:
If you see any removed `hasAssertion` call, I forgot to remove them during earlier PRs. I usually put them inside of to-do test cases to force them to fail (since an empty test case will pass)